packages/odyssey-react/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "@okta/odyssey-react-theme": "^0.15.0",
-    "@react-aria/focus": "3.5.0",
+    "@react-aria/focus": "3.8.0",
```

Comment:
We no longer need this dependency given we've removed `FocusScope`

Reply:
50687c3c7751d1b48ee71f2b1996e7563ea4bc6e